src/index.ts

```diff
@@ -27,6 +29,18 @@ function subscribe(el: HTMLElement): Subscription {
   }
 }
 
+function markElementWithConfigIfNeeded(el: HTMLElement, optionConfig?: OptionConfig) {
+  // eslint-disable-next-line no-console
+  console.log(optionConfig)
+
+  if (optionConfig?.pasteAsPlainText) {
+    // eslint-disable-next-line no-console
+    console.log(optionConfig?.pasteAsPlainText, 'HERE')
+
```

Comment:
Likewise 
```suggestion
```

Reply:
Oops this is embarrassing, absolutely 👍 will remove shortly

Reply:
Happens all the time haha no worries 👍